src/createEpicMiddleware.ts

```diff
@@ -109,5 +125,19 @@ export function createEpicMiddleware<
     epic$.next(rootEpic);
   };
 
+  epicMiddleware.sigterm = () => {
+    // sends sigterm to all currently running epics
+    store.dispatch({ type: SIGTERM });
+  };
+
+  epicMiddleware.kill = rootEpic => {
+    // this should only kill the given epic, alias of `complete`.
+  };
+
+  epicMiddleware.killall = () => {
+    // running epics could be stored into a list when calling `middleware.run`
+    // this method could flush that list and call `complete`
+  };
```

Comment:
`sigterm` could just work inside the redux life cycle, so all currently running epics could complete their current iteration.

`kill` and `killall` would just terminate everything